docs/authentication/social-connections/linkedin-oidc.mdx

```diff
@@ -1,88 +1,116 @@
 ---
-title: LinkedIn OpenID Connect
+title: Set up a social connection with LinkedIn OpenID Connect
 description: Learn how to set up a social connection with LinkedIn in your Clerk application.
 ---
 
-# LinkedIn
-
-Learn how to set up a social connection with LinkedIn in your Clerk application.
-
-To make the development flow as smooth as possible, Clerk uses preconfigured shared OAuth credentials and redirect URIs for development instances - no other configuration is needed. Simply navigate to the [Social Connections page](https://dashboard.clerk.dev/last-active?path=user-authentication/social-connections) in your Clerk Dashboard, toggle on the LinkedIn option, and click **Save**.
-
-In production instances, you must provide custom credentials, which includes generating your own Client ID and Client Secret using your LinkedIn Developer account. Don't worry, this guide will walk you through that process in just a few simple steps.
-
-## Before you start
-
-- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.dev/). For more information, check out our [Set up your application guide](/docs/quickstarts/setup-clerk).
-- You need to have a LinkedIn account. To create one, [click here](https://developer.linkedin.com/).
-
-## Configuration
+<TutorialHero 
+  beforeYouStart={[
+    {
+      title: 'Create a Clerk application in your Clerk Dashboard',
+      link: 'https://clerk.com/docs/quickstarts/setup-clerk',
+    },
+    {
+      title: 'Create a LinkedIn developer account',
+      link: 'https://developer.linkedin.com/',
+    },
+  ]}
+>
+- Create a LinkedIn application
+- Enable LinkedIn as a social connection
+- Set the Authorized Redirect URI in your LinkedIn application
+- Set the Client ID and Client Secret in your Clerk Dashboard
+- Enable OpenID Connect in your LinkedIn application
+</TutorialHero>
+
+## Introduction
```

Comment:
We could even change "## Introduction" to "## Configure LinkedIn connection for your development instance" and then have the steps with screenshots to show how to enable LinkedIn in the Dashboard.
And then change the following section "## Tutorial" and "## Configure LinkedIn connection for your production instance"

@rachelnabors what do you think!

Reply:
I see what you're talking about now! Yes, this seems like a good pattern to standardize here. Please do :)